client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -63,7 +63,15 @@
         };
 
         this.update = function update(dictionary, data, success, error) {
-            return api.save('dictionaries', dictionary, data).then(success, error);
+            var sendData = {};
+            angular.forEach(data, function(val, key) {
+                if (key !== 'content') {
+                    sendData[key] = val === null? val: val.toString();
+                } else {
+                    sendData[key] = val;
+                }
```

Comment:
It's not about the length per se, it's about how many execution paths you have to walk through in your head before you deduce the value of `sendData[key]` in all four possible cases.

As I said, to me personally at least a compact version is more readable, because it makes it very explicit in which case `val.toString()` is used and that `val` is used in all other cases.